unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/Camera/OverrideCinemachineAxisInput.cs

```diff
@@ -18,12 +20,14 @@ public struct AxisToMeasurableAction
     [SerializeField] private AxisToMeasurableAction[] axisToMeasurableActions;
     private Dictionary<string, InputAction_Measurable> cachedAxisToMeasurableActions;
     private InputSpikeFixer inputSpikeFixer;
+    private bool invertMouseY = false;
 
     private void Awake()
     {
         cachedAxisToMeasurableActions = axisToMeasurableActions.ToDictionary(x => x.axisName, x => x.measurableAction);
         CinemachineCore.GetInputAxis = OverrideGetAxis;
         inputSpikeFixer = new InputSpikeFixer(() => Cursor.lockState);
+        DataStore.i.camera.invertYAxis.OnChange += SetInvertYAxis;
```

Comment:
We should call `SetInvertYAxis` with the current value. It might have been set before the subscription.

Reply:
Done